src/state/slices/portfolioSlice/utils.ts

```diff
@@ -23,12 +23,14 @@ export type UtxoParamsAndAccountType = {
 // TODO(0xdef1cafe): these should be exported from caip2
 export const ethChainId = 'eip155:1'
 export const btcChainId = 'bip122:000000000019d6689c085ae165831e93'
+export const cosmosChainId = 'cosmos:cosmoshub-4'
```

Comment:
Do we want to add osmosis here too? `cosmos:osmosis-1`

Reply:
we can but i haven't test it.

Reply:
Probably can wait on it for now. We'll need chain-adapters work to include osmosis with supportedChainAdapters anyway.